src/main/java/store/model/receipt/PurchaseProducts.java

```diff
@@ -0,0 +1,39 @@
+package store.model.receipt;
+
+import store.model.Product;
+
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PurchaseProducts {
+    private final List<Product> purchaseProducts;
+
+    public PurchaseProducts(String input) throws IOException {
+        purchaseProducts = new ArrayList<>();
+        validate(input);
+    }
+
+    private void validate(String input) throws IOException {
+        String[] inputProducts = input.split(",", -1);
+        for (String product : inputProducts) {
+            Product pro = new Product(product);
```

Comment:
```suggestion
            Product product = new Product(inputProduct);
```
이렇게 네이밍한다면 헷갈리지도 않고 ``pro``라고 줄일지 않아도 되지 않아서 좋을 것 같아요!